incubator/ark-server/templates/crd.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: backups.ark.heptio.com
+  labels:
+    component: ark
```

Comment:
The chart doesn't have multiple components. I would just remove this label everywhere.

Reply:
ok, I just had left it as it was in the original files